crates/web-sys/src/features/gen_DomTokenList.rs

```diff
@@ -138,13 +138,34 @@ extern "C" {
     #[doc = ""]
     #[doc = "*This API requires the following crate features to be activated: `DomTokenList`*"]
     pub fn contains(this: &DomTokenList, token: &str) -> bool;
+    # [wasm_bindgen (method , structural , js_class = "DOMTokenList" , js_name = entries)]
+    #[doc = "The `entries()` method."]
+    #[doc = ""]
+    #[doc = "[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/entries)"]
+    #[doc = ""]
+    #[doc = "*This API requires the following crate features to be activated: `DomTokenList`*"]
+    pub fn entries(this: &DomTokenList) -> ::js_sys::Iterator;
+    # [wasm_bindgen (catch , method , structural , js_class = "DOMTokenList" , js_name = forEach)]
+    #[doc = "The `forEach()` method."]
+    #[doc = ""]
+    #[doc = "[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/forEach)"]
+    #[doc = ""]
+    #[doc = "*This API requires the following crate features to be activated: `DomTokenList`*"]
+    pub fn for_each(this: &DomTokenList, callback: &::js_sys::Function) -> Result<(), JsValue>;
```

Comment:
[`Array::for_each()`](https://docs.rs/js-sys/0.3.68/js_sys/struct.Array.html#method.for_each) takes a `&mut dyn FnMut()` instead, which also can encode types. Both ways have tradeoffs and ideally we could have both, so I'm happy to leave it as is because I think it aligns more with all the other APIs `web-sys` currently exposes.

Reply:
Probably be best to wait for demand first.
But I appreciate the offer!